library/src/main/java/nl/nos/imagin/ScrollHandler.kt

```diff
@@ -59,9 +64,22 @@ class ScrollHandler(
                 }
             })
 
+    private fun ImageView.leftEdgeIsVisible(): Boolean {
```

Comment:
Shouldn't this be `translationX == maxTranslationXScale`?

Reply:
That is only true if the image is not scaled. If the image is scaled(zoomed in) than translation==0 would be the image in the center cropped.